k8s/quickstart/agent-daemonset.yaml

```diff
@@ -26,10 +26,20 @@ spec:
           # from https://github.com/lqhl/wait-for-it
           image: cgr.dev/chainguard/wait-for-it
           args: ["-t", "30", "spire-server:8081"]
+          env:
+          - name: MY_NODE_NAME
+            valueFrom:
+              fieldRef:
+                fieldPath: status.podIP
```

Comment:
We shouldn't need to set this in the init container. This init container is running a wait-for-it image to check if SPIRE Server is up and available, and the wait-for-it image won't be looking for this environment variable.

Reply:
Thanks for that, Ryan. Have removed it. Do let me know if you would like me to squash the changes too.